<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminuserClass</class>
 <widget class="QWidget" name="adminuserClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>adminuser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/学院logo（jiangsu university jingjiang college）.png</normaloff>image/学院logo（jiangsu university jingjiang college）.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>721</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image/雪山.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>59</y>
     <width>1181</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 200);</string>
   </property>
   <widget class="QLabel" name="tip1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>欢迎您，管理员：</string>
    </property>
   </widget>
   <widget class="QLabel" name="admin_name">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>none</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>请选择您要使用的功能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_stuinfo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加学生信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_tchinfo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加教师信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_stuuser">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加学生账号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_tchuser">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加教师账号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_stupass">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>170</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除学生账号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_tchpass">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除教师账号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_stuinfo">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除学生信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_tchinfo">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>410</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除教师信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_topic">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加选题题目</string>
    </property>
   </widget>
   <widget class="QPushButton" name="see_stuinfo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查看学生信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="see_tchinfo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查看教师信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_topic">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除选题题目</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contect_dev">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>更多功能请点击此按钮联系开发者开发</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>510</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
